slices/ca-certificates-java.yaml

```diff
@@ -7,7 +7,7 @@ slices:
   data:
     essential:
       - ca-certificates_data
-      - libnss3_libs
```

Comment:
I was about to complain about this change :p but it seems ok cause the contents are the same. At least for now. Are you sure `data` only depends on `nss` and not the other libraries?

Reply:
The `ca-certificates-java_data` slice currently have a `cacerts` file and an empty directory. We do not need the `nss` libraries at all **directly**, but they are used by the `openjdk` slices.

Anyway, yeah, it should be alright to include only the `nss` slice here.

Reply:
Thanks for adding it back. And ok for the coming PR. Feel free to open it already